tests/ltp/patches/lstat02.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/testcases/kernel/syscalls/lstat/lstat02.c b/testcases/kernel/syscalls/lstat/lstat02.c
+index 69d607089..c71b244d9 100644
+--- a/testcases/kernel/syscalls/lstat/lstat02.c
++++ b/testcases/kernel/syscalls/lstat/lstat02.c
+@@ -21,6 +21,10 @@
+  *	many symbolic links encountered while traversing.
+  */
+ 
++/* Patch to remove the EFAULT related sub test as it currently causes
++ * enclave abort due to a product issue 169
++ */
++
+ #include <errno.h>
+ #include <pwd.h>
+ #include <stdlib.h>
+@@ -50,7 +54,6 @@ static struct test_case_t {
+ } test_cases[] = {
+ 	{TEST_EACCES, EACCES},
+ 	{TEST_ENOENT, ENOENT},
+-	{NULL, EFAULT},
```

Comment:
@ashokchandran , Instead of removing the EFAULT tests, comment it for now and add a TODO taks (for ex: Test will be enabled after issue 169 is fixed). Applies for all file changes